tests/EmailAddressValidatorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace ipl\Tests\Validator;
+
+use DateTime;
+use ipl\I18n\NoopTranslator;
+use ipl\I18n\StaticTranslator;
+use ipl\Validator\DateTimeValidator;
+use ipl\Validator\EmailAddressValidator;
+
+class EmailAddressValidatorTest extends TestCase
+{
+    public function testEmailAddressValidatorWithValidEmailAddress()
+    {
+        StaticTranslator::$instance = new NoopTranslator();
+        $this->assertTrue((new EmailAddressValidator())->isValid("test@test.com"), 'email address is valid');
```

Comment:
Messages are used if assertions fail. So if `test@test.com` is not valid, the reason for this test to fail, then the message reads `email address is valid`. It should read `valid email address is invalid` for example.

The same applies to the test below.